src/Microsoft.Azure.ServiceBus/Management/ManagementClient.cs

```diff
@@ -226,8 +226,15 @@ public Task<QueueDescription> CreateQueueAsync(string queueName, CancellationTok
 
         public async Task<QueueDescription> CreateQueueAsync(QueueDescription queueDescription, CancellationToken cancellationToken = default)
         {
+            queueDescription.NormalizeDescription(this.csBuilder.Endpoint);
             var atomRequest = queueDescription.Serialize().ToString();
-            var content = await PutEntity(queueDescription.Path, atomRequest, false, cancellationToken);
+            var content = await PutEntity(
+                queueDescription.Path, 
+                atomRequest, 
+                false, 
+                queueDescription.ForwardTo, 
+                queueDescription.ForwardDeadLetteredMessagesTo, 
+                cancellationToken);
```

Comment:
I'm assuming there'll be a pass to ensure all configureAwaits are in place.

Reply:
Haha.. I guess so

Reply:
@nemakam I know where you live